easybuild/easyblocks/c/cuda.py

```diff
@@ -75,7 +75,7 @@ def install_step(self):
         if 'DISPLAY' in os.environ:
             os.environ.pop('DISPLAY')
 
-        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True)
+        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True, maxhits=1500)
```

Comment:
Actually I think it was yesterday a local problem. Access to storage was very slow temporarily. I guess we can reduce it to let's say 300?